services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -216,6 +227,42 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
     dispatch(change(FORM.VIEW_EDIT_REPORT, "documents", docs));
   };
 
+  const fetchComments = async () => {
+    setIsLoading(true);
+    await dispatch(fetchMineReportComments(mineGuid, formValues.mine_report_guid));
+    setIsLoading(false);
+  };
+
+  useEffect(() => {
```

Comment:
can take out this useEffect and default the isLoading status to false- inspect the formValues to grab the comments property- can `fetchLatestReportSubmission` on add comment to have formValues be updated properly.

Reply:
I tried to used load comment section with formValues.comments initially and then to load it with fetchComments. However, once its assigned it always use the same source. Let's have a call on this please.

Reply:
As discussed, updated the backend to use mine report.